samples/zigbee/ncp/Kconfig

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if USB
+config USB_DEVICE_PID
+	default USB_PID_CDC_ACM_SAMPLE
+endif
+
+config NCP_FW_VERSION
```

Comment:
Please remove or use it inside the `ZB_WEAK_PRE zb_uint32_t ZB_WEAK zb_osif_get_fw_version(void)` function (`zb_nrf_platform.c`).

Reply:
removed